src/sources/mysql/mysql-schema.lisp

```diff
@@ -4,57 +4,29 @@
 
 (in-package :pgloader.source.mysql)
 
-;;;
-;;; Those functions are to be called from withing an already established
-;;; MySQL Connection.
-;;;
-;;; Handle MATERIALIZE VIEWS sections, where we need to create the views in
-;;; the MySQL database before being able to process them.
-;;;
-(defun create-my-views (views-alist)
-  "VIEWS-ALIST associates view names with their SQL definition, which might
-   be empty for already existing views. Create only the views for which we
-   have an SQL definition."
-  (unless (eq :all views-alist)
-    (let ((views (remove-if #'null views-alist :key #'cdr)))
-      (when views
-        (loop :for (name . def) :in views
-           :for sql := (destructuring-bind (schema . v-name) name
-                         (format nil
-                                 "CREATE VIEW ~@[~s.~]~a AS ~a"
-                                 schema v-name def))
-           :do
-           (log-message :info "MySQL: ~a" sql)
-           #+pgloader-image
-           (mysql-query sql)
-           #-pgloader-image
-           (restart-case
-               (mysql-query sql)
-             (use-existing-view ()
-               :report "Use the already existing view and continue"
-               nil)
-             (replace-view ()
-               :report "Replace the view with the one from pgloader's command"
-               (let* ((v-name   (cdr name))
-                      (drop-sql (format nil "DROP VIEW ~a;" v-name)))
-                 (log-message :info "MySQL: ~a" drop-sql)
-                 (mysql-query drop-sql)
-                 (mysql-query sql)))))))))
-
-(defun drop-my-views (views-alist)
-  "See `create-my-views' for VIEWS-ALIST description. This time we DROP the
-   views to clean out after our work."
-  (unless (eq :all views-alist)
-    (let ((views (remove-if #'null views-alist :key #'cdr)))
-      (when views
-        (let ((sql
-               (format nil "DROP VIEW ~{~a~^, ~};"
-                       (mapcar (lambda (qname)
-                                 (format nil "~@[~s.~]~a"
-                                         (car qname) (cdr qname)))
-                               (mapcar #'car views)))))
-          (log-message :info "MySQL: ~a" sql)
-          (mysql-query sql))))))
+(defclass copy-mysql (db-copy)
```

Comment:
Minor issue: if the superclass is named `db-copy` then the children should perhaps be named `mysql-copy`, `mssql-copy`, ...

`copy-mysql` sounds much more like a verb (function) than a noun (instance).

Reply:
Good thinking. It's too invasive for this PR though. Also I wonder if I should get rid of the COPY idea and rely more on the catalogs, where most of the work is done nowadays anyway.